src/api/v1beta1/properties.go

```diff
@@ -94,7 +94,7 @@ func (dk *DynaKube) NeedsStatsd() bool {
 	return dk.IsActiveGateMode(StatsdIngestCapability.DisplayName)
 }
 
-func (dk *DynaKube) HasActiveGateTLS() bool {
+func (dk *DynaKube) HasActiveGateCaCert() bool {
```

Comment:
Isn't it a bit confusing to call the field TLSSecret in the CR and the function `HasActiveGateCaCert()`?

Reply:
It's named wrong, `TLS != CA`. 
It's way more confusing (for me) to understand source code before this change, especially because we have multiple instances of TLS protocol related objects, e.g. CAs or private keys.
I agree naming in CRD shouldn't be changed, though.